07-multi-hop/example_test_guide.md

```diff
@@ -0,0 +1,106 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check multi-hop connectivity over IPv6
+
+### Pre-setup Steps
```

Comment:
We also need to mention that all applications (task 1-4) need to be compiled with `USEMODULE=gnrc_pktbuf_cmd`.